### ISSUE TITLE
COSE-Sign1-corim payload doesn't use global code point for unsigned-corim-map

### ISSUE THREAD

**Issue by nedmsmith** [ietf-rats/ietf-corim-cddl#114]
Should payload: bstr .cbor unsigned-corim-map in COSE-Sign1-corim us the global code point for unsigned-corim-map?

E.g., payload: bstr .cbor #6.501(unsigned-corim-map)

**Comment by thomas-fossati** [comment]
> Should payload: bstr .cbor unsigned-corim-map in COSE-Sign1-corim us the global code point for unsigned-corim-map?

Why? I don't see any ambiguity that would justify the extra tagging.

**Comment by nedmsmith** [comment]
Does a parser that only understands 506 be able to be invoked by the parser that only understands signed-corim-map be able to make the hand off without 506?

**Comment by thomas-fossati** [comment]
> Can a parser that only understands 506 take a hand off from a signed-corim parser that can't parse 506 or unsigned-corim?

yes, I think so:  an hypothetical CoRIMParser goes through unsigned-corim-map and finds out 506 in one of the comid.tags.  It starts CoMIDParser feeding it with the decap buffer and waits until the result are ready.
